.gitignore

```diff
@@ -4,4 +4,5 @@ venv/
 *.sqlite3
 build/
 *.egg-info
-staticfiles/
\ No newline at end of file
+staticfiles/
+.vscode/
```

Comment:
Dit is typisch zo'n regel om in je [global git ignore](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) te zetten!

Reply:
Ai ai kapitein!